README.md

```diff
@@ -32,13 +34,102 @@ Go 1.5 users will need to set `GO15VENDOREXPERIMENT=1` to get vendored dependenc
 Upon successful build the plugin binary will be available in `build/sriov`. 
 
 ## Enable SR-IOV
-
+### Intel cards
 Given Intel ixgbe NIC on CentOS, Fedora or RHEL:
 
 ```
 # vi /etc/modprobe.conf
 options ixgbe max_vfs=8,8
 ```
+### Mellanox cards
+SRIOV-CNI support Mellanox ConnectX�-4 Lx and ConnectX�-5 adapter cards.
+To enable SR-IOV functionality the following steps are required:
+
+1- Enable SR-IOV in the NIC's Firmware.
+
+> Installing Mellanox Management Tools (MFT) or mstflint is a pre-requisite, MFT can be downloaded from [here](http://www.mellanox.com/page/management_tools), mstflint package available in the various distros and can be downloaded from [here](https://github.com/Mellanox/mstflint).
+
+Use Mellanox Firmware Tools package to enable and configure SR-IOV in firmware
+
+```
+# mst start
```

Comment:
@Mmduh-483 
I'm trying to configure a ConnectX-4 using this guide on CentOS 7(update kernel 4.4.170-1.el7.elrepo.x86_64). I've managed to install  mstflint. But looks like it didn't install the 'mst' tool that is shown in subsequent steps. Am I missing something?

Reply:
So, the example above is valid when using MFT.

for mstflint:
1. no need to do mst start
2. use mstconfig instead of mlxconfig and provide the PCI device address as the argument for '-d' flag.

ill submit a PR to clarify later today.

Reply:
@adrianchiris 
Thanks for clarifying it. It's worth clearing this up in this doc. OR at least add some external links showing the steps required for initializing the NICs and how to create VFs with different options(i.e. some advance options such as shared port etc. if possible)